@xen-orchestra/web-core/lib/components/tree/TreeLine.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div class="tree-line">
+    <div :class="{ 'tree-line-half-height': halfHeight }" class="tree-line-vertical" />
+    <div :class="{ right }" class="tree-line-horizontal" />
+  </div>
+</template>
+
+<script lang="ts" setup>
+defineProps<{
+  halfHeight?: boolean
+  right?: boolean
+}>()
+</script>
+
+<style lang="postcss" scoped>
+.tree-line {
+  flex: 0 1 1.25em;
```

Comment:
It seems that a 0.7rem offset is closer to the DS.

Also, I suggest a variable to handle that.

```suggestion
.tree-line {
  --offset: 0.7rem;
  flex: 0 1 1.25em;
```